micronota/bfillings/diamond.py

```diff
@@ -230,20 +231,22 @@ def _annotate_fp(self, fp, aligner='blastp', evalue=0.001, cpus=1,
             elif outfmt == 'sam':
                 res = res.append(
                     self._filter_id_cov(self.parse_sam(out_fp)))
-            found.extend(res.index)
             # save to a tmp file the seqs that do not hit current database
             new_fp = join(self.tmp_dir, '%s.fa' % out_prefix)
+            num_hits = 0
             with open(new_fp, 'w') as f:
                 for seq in read(fp, format='fasta'):
                     if seq.metadata['id'] not in found:
                         seq.write(f, format='fasta')
                         seqs.append(seq)
+                        num_hits += 1
+            logger.info('Number of diamond hits: %d' % num_hits)
```

Comment:
The `num_hits` is the number of seq left unmatched from last run. how about changing log to something like `XXX seqs are left unmatched`?